RNAndroidPlayground/app/src/main/java/com/kiwi/rnandroidplayground/HotelFragment.kt

```diff
@@ -0,0 +1,36 @@
+package com.kiwi.rnandroidplayground
+
+import android.os.Bundle
+import com.facebook.react.ReactNativeHost
+import com.kiwi.rnkiwimobile.RNKiwiFragment
+
+class HotelFragment : RNKiwiFragment() {
+
+  override fun getModuleName(): String {
+    return "NewKiwiHotels"
+  }
+
+  override fun getReactNativeHost(): ReactNativeHost = (activity.application as PlaygroundApplication).reactNativeHost
+
+  // TODO extract code repetition
+  override fun getInitialProperties(): Bundle? {
```

Comment:
I would rather use POJO (POKO actually :-)) for defining the data scructure than Bundle to be type safe.